prereq.sh

```diff
@@ -348,32 +370,11 @@ echo -e "${NC}"
 # Show SELinux reboot message if necessary
 if [[ "$SELINUX_REBOOT_REQUIRED" == "true" ]]; then
     echo -e "${RED}"
-    echo "SELinux has been disabled. Please reboot the system, unless it is an upgrade or restore scenario, before proceeding with the playbook execution !"
+    echo "SELinux has been successfully disabled. Please reboot the system before proceeding. However, if you are upgrading or restoring the control plane, avoid rebooting to prevent the loss of telemetry data."
     echo -e "${NC}"
 fi
 
 # Check if the OS version is unsupported and print a warning message
```

Comment:
this comment should be moved with the code block to line 168

Reply:
noted